### PR TITLE
Allow for analysis under rebar3_hank

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

File: elvis.config [hunk 1]
```diff
@@ -38,11 +38,6 @@
                  ],
         ignore => [ktn_dodger]
        },
-      #{dirs => ["."],
```
Comment (paulo-ferraz-oliveira) [inline elvis.config:41]: `rebar3_lint` told me this wasn't being used.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ${{matrix.os}}
     strategy:
       matrix:
-        otp_vsn: [20, 21, 22, 23, 24]
```
Comment (paulo-ferraz-oliveira) [inline .github/workflows/ci.yml:16]: `rebar3_hank` is 21+

File: rebar.config [hunk 3]
```diff
@@ -43,6 +43,19 @@
...
+]}.
 
 {alias, [{test, [lint, xref, dialyzer, ct, cover]}]}.
+
+%% == hank ==
+
+{hank, [
+    {ignore, [
+        {"example/**", unnecessary_function_arguments},
+        {"test/**", unnecessary_function_arguments}
```
Comment (elbrujohalcon) [inline rebar.config:59]: This shouldn't be necessary. Just adding `-behavior(ct_suite).` to the test modules should be enough.

**Review by elbrujohalcon — CHANGES_REQUESTED** [review]

File: rebar.config (hunk 3 above)
Comment (paulo-ferraz-oliveira) [reply rebar.config:59]: This is under CI: that behaviour is not 21-compatible. Do you prefer a macro, there?

Comment (elbrujohalcon) [reply rebar.config:59]: Replied at https://github.com/project-fifo/rebar3_lint/pull/37#discussion_r676310071

Comment (paulo-ferraz-oliveira) [reply rebar.config]: Pushed.

File: rebar.config [hunk 4]
```diff
@@ -16,6 +16,8 @@
            , warn_untyped_record
            , debug_info]}.
 
+{minimum_otp_vsn, "21"}.
```
Comment (paulo-ferraz-oliveira) [inline rebar.config:19]: Because we're using macro `OTP_RELEASE`.

File: src/ktn_dodger.erl [hunk 5]
```diff
@@ -86,22 +86,19 @@
 -define(pp_form, '?preprocessor declaration?').
 
 
-%% @type errorinfo() = {ErrorLine::integer(),
```
Comment (paulo-ferraz-oliveira) [inline src/ktn_dodger.erl:89]: All EDoc-related changes courtesy of `rebar3 edoc`'s results.

**Review by elbrujohalcon — APPROVED** [review]

**Pull request merged**